README.md

```diff
@@ -96,6 +96,7 @@ Before running tangle-accelerator, please edit binding address/port of accelerat
 * `TA_PORT`: port of accelerator instance
 * `IRI_HOST`: binding address of IRI
 * `IRI_PORT`: port of IRI
+* `TA_THREAD`: Number of threads in the thread pool to process HTTP connections. The default number is 10.
```

Comment:
I don't fully understand why it defaults to 10. Can you explain?

Reply:
I do not modify the default number of thread, which is defined in `config.h` before this PR.
Should it default to a smaller number(1,2,4,8) for basic performance?
Users could change the thread numbers according to their own system environment.

Reply:
The consideration between thread pool and available native threads is different. We usually assign a bit fewer CPUs for thread pool, so that we can have better responsiveness.

Reply:
Since MHD does not provide a feature to dynamically adjust the number of threads,
Does it default to 4 reasonable?. Or do we need to get the number of CPUs by `get_nprocs_conf() or get_nprocs()` and set it to a certain proportion of the CPU number?

We actually use the option `TA_THREAD` nowhere before, so I'd like to amend the naming and description.

Reply:
Let's create a new issue which concentrates on thread assignment.